templates/flipflop-deployment.yaml

```diff
@@ -30,7 +30,7 @@ spec:
            "--log-level",
            "debug",
            "--fault-injection",
-           "--replica-count=1",
+           "--dry-run",
```

Comment:
Is this intended or a left-over of your debug setup?

Reply:
intended. replica-count is a config var as well, so it doesnt need to be a flag. I will be removing the flag from flipflop in this [pr](https://github.com/metal-toolbox/flipflop/pull/6)